pyrobosim/pyrobosim/core/world.py

```diff
@@ -607,6 +607,7 @@ def graph_node_from_entity(self, entity_query, resolution_strategy="nearest"):
             if entity is None:
                 entity = resolve_to_location(self, category=entity_query,
                     expand_locations=True, resolution_strategy=resolution_strategy)
+            # If not able to resolve to location, try to resolve to object
```

Comment:
Will add a variation of this comment, as this seems like the one new thing that this PR has.